isis/src/mex/objs/HrscCamera/HrscCamera.cpp

```diff
@@ -119,14 +119,22 @@ namespace Isis {
 
 
   /**
-   * @param filename
+   * @param cube
    */
-  void HrscCamera::ReadLineRates(QString filename) {
-    Table timesTable("LineScanTimes", filename);
+  void HrscCamera::ReadLineRates(Cube &cube) {
+    Table timesTable("LineScanTimes");
+
+    // check if LineScanTimes was added from isd
+    if (cube.hasTable("LineScanTimes")) {
+      timesTable = cube.readTable("LineScanTimes");
```

Comment:
Can use this line in both cases?

Reply:
Yeah I think you're right. This has been updated.